health-monitor/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "health-monitor",
+  "version": "0.0.1",
+  "description": "Healthcheck Monitoring for polkassembly",
+  "main": "index.js",
+  "scripts": {
+    "start": "ts-node src/index.ts",
```

Comment:
Should we have a `start:prod` script to run the transpiled js cc https://github.com/TypeStrong/ts-node/issues/104#issuecomment-337144823

Reply:
done